src/pages/Dashboard/Modals/SupplyWithdraw/SupplyWithdrawForm.tsx

```diff
@@ -164,10 +200,23 @@ interface ISupplyWithdrawFormProps extends ISupplyWithdrawFormUiProps {
   onSubmit: IAmountFormProps['onSubmit'];
 }
 
-const SupplyWithdrawForm: React.FC<ISupplyWithdrawFormProps> = ({ onSubmit, ...props }) => (
-  <AmountForm onSubmit={onSubmit}>
-    {formikBag => <SupplyWithdrawContent {...props} {...formikBag} />}
-  </AmountForm>
-);
+const SupplyWithdrawForm: React.FC<ISupplyWithdrawFormProps> = ({
+  onSubmit,
+  maxInput,
+  ...props
+}) => {
+  const onSubmitHandleError: IAmountFormProps['onSubmit'] = async (value: string) => {
+    try {
+      await onSubmit(value);
+    } catch (err) {
+      toast.error({ title: (err as Error).message });
```

Comment:
So to follow our pattern, I think this should throw a `ToastError` which gets caught by `SupplyWithdrawContent` to render the toast.

Reply:
This does follow our pattern in that the error is displayed from a UI component. This is not a container that contains side effects.

Reply:
You're right 👍